proposals/dao/tip_121c.ts

```diff
@@ -78,6 +124,75 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
     '(millions)'
   );
   console.log('simpleFeiDaiPSM FEI balance', (await contracts.fei.balanceOf(addresses.simpleFeiDaiPSM)) / 1e18);
+
+  // 1. Verify old DAI PSM is deprecated
+  // Has no funds
+  expect(await contracts.fei.balanceOf(addresses.daiFixedPricePSM)).to.equal(0);
+  expect(await contracts.dai.balanceOf(addresses.daiFixedPricePSM)).to.equal(0);
+
+  // Fully paused
+  expect(await contracts.daiFixedPricePSM.paused()).to.be.true;
+  expect(await contracts.daiFixedPricePSM.redeemPaused()).to.be.true;
+  expect(await contracts.daiFixedPricePSM.mintPaused()).to.be.true;
+
+  // 2. Verify new DAI PSM has DAI and no FEI (should have been burned)
+  // DAI on PSM should cover the user circulating supply of FEI
+  expect(await contracts.dai.balanceOf(addresses.simpleFeiDaiPSM)).to.be.bignumber.greaterThan(
+    userCirculatingFeiSupply
+  );
```

Comment:
nit should be geq

Reply:
Ideally we can get a strict equality at least against a specific fork block

Reply:
These are chai assertions - geq is a BigNumber type specific condition check. Chai doesn't have a geq

Reply:
Pushed equality check